### ISSUE TITLE
Feature-Request: Set modes and temperatures

### ISSUE THREAD

**Issue by optimismus** [CBrosius/luxtronik_v1#14]
I would like to set "WP_Modus_Heizung", "WP_Modus_Warmwasser", "WP_Temp_Brauchwasser_Soll_Temperatur" and "WP_Temp_Heizkreis_Soll_Ruecklauftemperatur".
That would help to use photovoltaik engery or dynamic energy costs (e.g. Tibber prices) when they are cheap(er).

**Comment by CBrosius** [comment]
Yes, that is the next step. But we have to send several commands with the right timing to achive this.

**Comment by optimismus** [comment]
Thank you. We had a private discussion on this, but I thought github is the place the information should be.

BTW: You can use labels (here "enhancement") to flag issues.

BTW2: I would appreciate, if you enable discussion for this repro.

**Comment by CBrosius** [comment]
Great idea, opened Discussion-Area directly. And thank you for your great documentation-additions.

**Comment by CBrosius** [comment]
A few minutes ago I uploaded a new ESP32-Version in the WorkInProgress-branch with the first implementation of the Set-Possibility.
You are able to set Heizungsmode and Warmwassermode.

Please let me know how this is working for you.

**Comment by optimismus** [comment]
Thank you very much for your work.
I tried both Heizungsmode and Warmwassermode, when the heatpump was doing "Brauchwasser". It worked for Warmwassermode, Heizungsmode is reset after a second:
![grafik](https://github.com/CBrosius/luxtronik_v1/assets/16720046/9d67098f-5bda-45e1-8759-a9f03c3e3586)
To set heating on/off will be quite cool to use solar power or cheap engery costs (e.g. Tibber). But a more fine tuning possibility would be to set the SOLL-temperatures.

**Comment by ohosch** [comment]
Is still someone working on this? 
I am Using this great integration since around 2 Month. This Morning something got corrupt, so I could only get back by power-cycling the HeatPump. Unfortunately I did not capture the esp output. I think it got stuck in programming mode and could not get out of it. It looked like pressing e.g. "uart Exit ProgramMode" did not send any string to the HeatPump.  
Is it somehow possible to get logs directly from the ESP?

**Comment by cbrosius** [comment]
I still use it as it is, but have not done any further development.
For me, reading the values is enough, I don´t need to set modes or write anything to the heatpump.

**Comment by cbrosius** [comment]
Hier mal ein Link zur Umfrage ob ich das Schreiben von Werten einbauen soll: https://github.com/cbrosius/luxtronik_v1/discussions/30
